app/javascript/react/components/molecules/Calendar/HeaderToggle/HeaderToggle.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import { screenSizes } from "../../../../utils/media";
+import { Container, RotatedIcon, Heading } from "./HeaderToggle.style";
+import arrowIcon from "../../../../assets/icons/arrowIcon.svg";
+
+interface Props {
+  titleText: string;
+  componentToToggle: JSX.Element;
+}
+
+const HeaderToggle: React.FC<Props> = ({ titleText, componentToToggle }) => {
+  const [isVisible, setIsVisible] = useState<boolean>(true);
+  const [isRotated, setIsRotated] = useState<boolean>(false);
```

Comment:
Could we use only one state` isVisible` or `isRotated`?

Reply:
Yes, true! Thank you :)